static/css/main.css

```diff
@@ -81,6 +82,7 @@ a {
 .banner-info .apply {
   margin-top: 40px;
   display: none;
+  text-align: right;
```

Comment:
I think maybe we should just keep everything centered on mobile, including the HackNC text, countdown, and banner graphic. The graphics could also be a bit bigger and centered at the top